### PR TITLE
Refactor: rename variables camel case

### LINKED ISSUE #466: Rename variables to camelCase
### Objective
We want to rename all the variables used in the project to camelCase and make sure they have meaningful names

### Acceptance Criteria
- Rename all variables used in the project to camelCase and make sure they have meaningful names.
- Don't rename anything else which is not a variable. Functions, class names, files names are out of scope for this ticket
- Make sure that the test run and nothing breaks

### PR DESCRIPTION
**Pull request by Kingcedru**
### Describe your changes

Changed all variables to camelCase.

### Issue ticket number and link

- #466 

### Checklist before requesting a review

- [ ] Rename all variables used in the project to camelCase and make sure they have meaningful names.
- [ ] Don't rename anything else which is not a variable. Functions, class names, files names are out of scope for this ticket
- [ ] I have performed a self-review of my code
- [ ] No existing features have been broken without good reason.
- [ ] The code style guideline have been followed
- [ ] Documentation has been updated to reflect your changes.

### CODE REVIEW

File: apps/drec-api/src/pods/access-control-layer-module-service/access-control-layer-module-service.spec.ts [hunk 1]
```diff
@@ -11,7 +11,7 @@ import { ConflictException } from '@nestjs/common';
 describe('AccessControlLayerModuleServiceService', () => {
   let service: AccessControlLayerModuleServiceService;
   let repository: Repository<AClModules>;
-  let Permissionvalue: DecimalPermissionValue;
+  let PermissionValue: DecimalPermissionValue;
```
Comment (musayann) [inline apps/drec-api/src/pods/access-control-layer-module-service/access-control-layer-module-service.spec.ts]: variables should be in camelCase no ?

File: apps/drec-api/src/pods/certificate-log/certificate-log.service.spec.ts [hunk 2]
```diff
@@ -331,15 +331,15 @@ describe('CertificateLogService', () => {
         ],
       };
 
-      const getnewreservationinfo = { deviceGroups: [] };
+      const getNewReservationInfo = { deviceGroups: [] };
```
Comment (musayann) [inline apps/drec-api/src/pods/certificate-log/certificate-log.service.spec.ts]: `get` should be used on functions only not on variables

Comment (musayann) [reply apps/drec-api/src/pods/certificate-log/certificate-log.service.spec.ts]: remove the `new` keyword as well

**Review by musayann — CHANGES_REQUESTED** [review]

File: apps/drec-api/src/pods/access-control-layer-module-service/common/permissionBitposition.ts [hunk 3]
```diff
@@ -110,7 +110,7 @@ export class DecimalPermissionValue {
         ).isSet = true;
       }
     });
-    const getpermission: any = [];
+    const permission: any = [];
```
Comment (musayann) [inline apps/drec-api/src/pods/access-control-layer-module-service/common/permissionBitposition.ts]: this should be named `permissions`

File: apps/drec-api/src/pods/admin/admin.controller.ts [hunk 4]
```diff
@@ -320,35 +320,35 @@ export class AdminController {
     if (!user) {
       throw new NotFoundException('Does not exist');
     }
-    const manyotheruserinorg =
+    const manyOtherUserInOrg =
```
Comment (musayann) [inline apps/drec-api/src/pods/admin/admin.controller.ts]: I would suggest a name like `otherOrgUsers`

File: apps/drec-api/src/pods/certificate-log/certificate-log.service.spec.ts [hunk 5]
```diff
@@ -331,15 +331,15 @@ describe('CertificateLogService', () => {
         ],
       };
 
-      const getnewreservationinfo = { deviceGroups: [] };
+      const reservationInfo = { deviceGroups: [] };
 
-      const getReservationInforDeveloperBsiseSpy = jest
+      const reservationInforDeveloperBsiseSpy = jest
```
Comment (musayann) [inline apps/drec-api/src/pods/certificate-log/certificate-log.service.spec.ts]: this variable name looks strange

File: apps/drec-api/src/pods/certificate-log/certificate-log.service.spec.ts [hunk 6]
```diff
@@ -449,9 +449,9 @@ describe('CertificateLogService', () => {
 
       const pageNumber = 1;
 
-      const getoldreservationinfo = { deviceGroups: [] };
+      const getOldReservationInfo = { deviceGroups: [] };
```
Comment (musayann) [inline apps/drec-api/src/pods/certificate-log/certificate-log.service.spec.ts]: shouldn't this be name `oldReservationInfo`

File: apps/drec-api/src/pods/certificate-log/certificate-log.service.spec.ts [hunk 7]
```diff
@@ -449,9 +449,9 @@ describe('CertificateLogService', () => {
...
 
-      const getnewreservationinfo = {
+      const getNewReservationInfo = {
```
Comment (musayann) [inline apps/drec-api/src/pods/certificate-log/certificate-log.service.spec.ts]: and this to `reservationInfo`

**Review by musayann — CHANGES_REQUESTED** [review]

File: apps/drec-api/src/pods/device/device.controller.ts [hunk 8]
```diff
@@ -773,20 +773,20 @@ export class DeviceController {
...
+    if (checkIsUnGroup) {
+      let fitleRop: any;
       if (role === 'Admin') {
-        fitlerop = {
+        fitleRop = {
           groupId: null,
         };
       } else {
-        fitlerop = {
+        fitleRop = {
```
Comment (musayann) [inline apps/drec-api/src/pods/device/device.controller.ts]: what does `fitleRop` I think there is a typo here

File: apps/drec-api/src/pods/email-confirmation/email-confirmation.service.ts [hunk 9]
```diff
@@ -337,8 +337,8 @@ export class EmailConfirmationService {
 
   async remove(userId: number): Promise<void> {
     this.logger.verbose(`With in remove`);
-    const allemialconfirm = await this.get(userId);
-    await this.repository.delete(allemialconfirm.id);
+    const allemialConfirm = await this.get(userId);
```
Comment (musayann) [inline apps/drec-api/src/pods/email-confirmation/email-confirmation.service.ts]: what does `allemial` mean ? I think they meant `allEmailConfirmations`. I would suggest to just use `confirmation`

File: apps/drec-api/src/pods/invitation/invitation.service.spec.ts [hunk 10]
```diff
@@ -436,21 +436,21 @@ describe('InvitationService', () => {
         .spyOn(userService, 'findByEmail')
         .mockResolvedValueOnce(mockApiUserEntity as IUser)
         .mockResolvedValueOnce(null);
-      const orgfindOneSpy = jest
+      const orgFindOneSpy = jest
         .spyOn(organizationService, 'findOne')
         .mockResolvedValue(inviteeOrganization as unknown as Organization);
-      const inviteefindOneSpy = jest
+      const inviteeFindOneSpy = jest
```
Comment (musayann) [inline apps/drec-api/src/pods/invitation/invitation.service.spec.ts]: let's rename it to `inviteFindOneSpy`

File: apps/drec-api/src/pods/permission/permission.service.spec.ts [hunk 11]
```diff
@@ -12,7 +12,7 @@ import { DecimalPermissionValue } from '../access-control-layer-module-service/c
 describe('PermissionService', () => {
   let service: PermissionService;
   let repository: Repository<ACLModulePermissions>;
-  let ACLpermissionService: AccessControlLayerModuleServiceService;
+  let ACLPermissionService: AccessControlLayerModuleServiceService;
```
Comment (musayann) [inline apps/drec-api/src/pods/permission/permission.service.spec.ts]: this should be `aclPermissionService`

File: apps/drec-api/src/pods/permission/permission.service.ts [hunk 12]
```diff
@@ -71,7 +71,7 @@ export class PermissionService {
       permissionValue,
     );
     if (permissionboolean) {
-      const aclpermission = new ACLModulePermissions({
+      const ACLPermission = new ACLModulePermissions({
```
Comment (musayann) [inline apps/drec-api/src/pods/permission/permission.service.ts]: This should `aclPermissionService`

File: apps/drec-api/src/pods/permission/permission.service.ts [hunk 13]
```diff
@@ -107,12 +107,12 @@ export class PermissionService {
       id: data.aclmodulesId,
     });
 
-    const Ispermission =
+    const isPermission =
```
Comment (musayann) [inline apps/drec-api/src/pods/permission/permission.service.ts]: This should `permissions`

File: apps/drec-api/src/pods/permission/permission.service.ts [hunk 14]
```diff
@@ -170,7 +170,7 @@ export class PermissionService {
   }
   async FindbyRole(id: number): Promise<IACLmodulsPermissions[]> {
     this.logger.verbose(`With in FindbyRole`);
-    const aclpermission = await this.repository.find({
+    const ACLPermission = await this.repository.find({
```
Comment (musayann) [inline apps/drec-api/src/pods/permission/permission.service.ts]: This should `aclPermission`

File: apps/drec-api/src/pods/permission/permission.service.ts [hunk 15]
```diff
@@ -217,19 +217,19 @@ export class PermissionService {
...
-      aclmodulesId: userpermission.aclmodulesId,
-      permissions: data.permissions,
-    };
-    const permissionboolean = await this.checkForExistingmodulepermission(
-      checkdata,
+    // const checkData = {
+    //   aclmodulesId: userPermission.aclmodulesId,
+    //   permissions: data.permissions,
+    // };
+    const permissionBoolean = await this.checkForExistingmodulepermission(
```
Comment (musayann) [inline apps/drec-api/src/pods/permission/permission.service.ts]: This should be `hasPermission`

File: apps/drec-api/src/pods/permission/permission.service.ts [hunk 16]
```diff
@@ -282,29 +282,29 @@ export class PermissionService {
       this.logger.error(`No module permission available in requeste`);
       throw new NotFoundException(`No module permission available in requeste`);
     }
-    const api_user = await this.userService.findById(loginuser.id);
+    const apiUser = await this.userService.findById(loginuser.id);
 
     let permissionIds: any = [];
-    const api_userpermission = await this.userService.getApiuser(
-      api_user.api_user_id,
+    const apiUserpermission = await this.userService.getApiuser(
```
Comment (musayann) [inline apps/drec-api/src/pods/permission/permission.service.ts]: This should `apiUserPermission`

File: apps/drec-api/src/pods/sdgbenefit/sdgbenefit.controller.ts [hunk 17]
```diff
@@ -63,7 +63,7 @@ export class SdgbenefitController {
   })
   getFuelTypes(): SDGBCodeNameDTO[] {
     this.logger.verbose(`With in getFuelTypes`);
-    const sdgbcode = this.SdgbenefitService.getSDGBCode();
-    return sdgbcode.map((sdgb) => plainToClass(SDGBCodeNameDTO, sdgb));
+    const SDGBCode = this.SdgbenefitService.getSDGBCode();
+    return SDGBCode.map((sdgb) => plainToClass(SDGBCodeNameDTO, sdgb));
```
Comment (musayann) [inline apps/drec-api/src/pods/sdgbenefit/sdgbenefit.controller.ts]: This should be `sdgBenefitCode`

File: apps/drec-api/src/pods/device-group/device-group.service.ts [hunk 18]
```diff
@@ -2565,8 +2565,8 @@ export class DeviceGroupService {
               );
             }
             if (filterDto.SDGBenefits) {
-              const newsdg = filterDto.SDGBenefits.toString();
-              const sdgBenefitsArray = newsdg.split(',');
+              const newSdg = filterDto.SDGBenefits.toString();
```
Comment (musayann) [inline apps/drec-api/src/pods/device-group/device-group.service.ts]: this should be `newSDG`

File: apps/drec-api/src/pods/device-group/device-group.service.ts [hunk 19]
```diff
@@ -2839,8 +2839,8 @@ export class DeviceGroupService {
...
```
Comment (musayann) [inline apps/drec-api/src/pods/device-group/device-group.service.ts]: same here

File: apps/drec-api/src/pods/device-group/device-group.service.ts [hunk 20]
```diff
@@ -2888,11 +2888,11 @@ export class DeviceGroupService {
         );
     });
     const totalCountQuery = await queryBuilder.getRawMany();
-    const groupedDatasql = await queryBuilder
+    const groupedDataSql = await queryBuilder
```
Comment (musayann) [inline apps/drec-api/src/pods/device-group/device-group.service.ts]: this should be `groupedDataSQL`

**Review by musayann — CHANGES_REQUESTED** [review]

**Review by musayann — DISMISSED** [review]

**Review by divinecharlotte — APPROVED** [review]

**Pull request merged**